src/environments/environment.local.ts

```diff
@@ -4,14 +4,15 @@
 import { IEnvironment } from "./environment.interface";
 export const environment: IEnvironment = {
   production: false,
-  host: `api/`,
-  siteIds: [],
+  siteIds: ["UA-40570867-6"],
+  hotJarId: 1662399,
```

Comment:
should our local machines be using the same `hotJarId` and `hotJarSv` as other environments? im guessing we dont have unique values for each environment yet as our environments are setup yet?

Reply:
no dev hotjar or analytics variables. only protected with env.production from loading. as in prev diffs. moved to variable rather than hard coded

Reply:
fair enough, better to have variable for sure. i imagine we will get the environment specific values once the actual environments are ready